features/step_definitions/breadcrumbs_steps.rb

```diff
@@ -0,0 +1,14 @@
+Given(/^the page exists$/) do
+  stub_content_store_has_item("/search", schema: "special_route")
+end
+
+When(/^I visit the page$/) do
```

Comment:
```ruby
When(/^I visit {string}$/) do |path|
  visit path
end
```

Reply:
Or instead of doing all of that you can simply reuse an existing step, please, see a comment above.